contrib/python/README.md

```diff
@@ -7,9 +7,33 @@ See [libpod](https://github.com/projectatomic/libpod)
 
 ###
 
-To build the podman wheel:
+To build the podman egg:
 
 ```sh
 cd ~/libpod/contrib/pypodman
-python3 setup.py clean -a && python3 setup.py bdist_wheel
+python3 setup.py clean -a && python3 setup.py bdist
+```
+
+Code snippets/examples:
+
+```python
+"""Show images in storage."""
+import podman
+
+with podman.Client() as client:
+  list(map(print, client.images.list()))
+```
+
+```python
+"""Show containers created since midnight."""
+from datetime import datetime, time
+import podman
+
+midnight = datetime.combine(datetime.today(), time.min, tzinfo=timezone.utc)
+with podman.Client() as client:
+  for c in client.containers.list():
+    created_at = podman.datetime_parse(c.createdat)
+
+    if created_at > midnight:
+      print("Ctnr {}: {}".format(c.id))
```

Comment:
This is a style nit, but I think most of out doc goes with either 'ctr' or 'container' and not 'ctnr'.  For examples like this I like to be abundantly clear and would prefer it spelled out as "Container"